crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -14,18 +16,54 @@ use super::{
 use crate::server::sui_sdk_data_provider::SuiClientLoader;
 use crate::{server::context_ext::DataProviderContextExt, types::base64::Base64};
 
-#[derive(Clone, Eq, PartialEq, Debug)]
+#[derive(Clone, Eq, PartialEq, Debug, SimpleObject)]
+#[graphql(complex)]
 pub(crate) struct Object {
     pub address: SuiAddress,
     pub version: u64,
     pub digest: String,
     pub storage_rebate: Option<BigInt>,
-    pub owner: Option<SuiAddress>,
+    pub owner: Option<Owner>,
```

Comment:
I don't think you want to make this change, because `Owner` *is* a complex edge.